### ISSUE TITLE
Failed to allocate 18446744073709551596 bytes

### ISSUE THREAD

**Issue by AkshajD** [poruloh/Eagle#37]
I tried running the command
```
./Eagle_v2.4.1/eagle --vcf=./file.vcf \
--geneticMapFile=./Eagle_v2.4.1/tables/genetic_map_hg19_withX.txt.gz \
--outPrefix=file.phased \
--outputUnphased
--chrom=1
```

However, I got a failed run, with it first running some batches, but after batch 10 of 10, it outputs the following:
```
WARNING: Sample 1 (1-indexed) has a het count of 0
ERROR: Failed to allocate 18446744073709551596 bytes
```

Any idea how this can be done?

**Comment by poruloh** [comment]
I believe this error arises when trying to phase a single sample without using a reference panel. You will need to use Eagle in reference-based phasing mode (e.g., using the 1000 Genomes reference panel) in order to phase a single sample.

**Comment by AkshajD** [comment]
So I am now trying it with the following command:
```
./Eagle_v2.4.1/eagle \
--vcfTarget=./file.vcf.gz \
--vcfRef=./1000G/ALL.chr1.phase3_shapeit2_mvncall_integrated_v5b.20130502.genotypes.vcf.gz \
--geneticMapFile=./Eagle_v2.4.1/tables/genetic_map_hg19_withX.txt.gz \
--outPrefix=file.phased \
--vcfOutFormat=z \
--outputUnphased \
--chrom=1
```
However, this returned a few errors talking about how there were M=0 SNPs to analyze in both target and reference, while 5315318 SNPs were ignored in target but not in reference and 6468094 SNPs were ignored in reference but not target. It also said there were 0 multi-allelic SNPs in target.
So, with this, it ends with `ERROR: Target and ref have too few matching SNPs (M=0)`

I first thought this might be because the 1000G VCF had the chromosome format without the chr prefix, while my sample had the chr prefix. However, after editing the 1000G VCF to incorporate that prefix, the error persists.

Suggestions?

**Comment by poruloh** [comment]
Is your sample VCF in hg38? If so, you would need to use a reference panel in the same build.

**Comment by AkshajD** [comment]
No, the VCF is on HG19. It just uses the version of HG19 that doesn't have chr (which I believe is called GRCh37, but I may be mistaken).
